pkg/mkimage-raw-efi/make-raw

```diff
@@ -370,7 +370,7 @@ done
 
 # Create a hybrid MBR to allow booting on legacy BIOS PC systems and ARM boards that
 # look for bootloaders in the first entry of the MBR
-sgdisk -h"${PROTECTIVE_MBR_LIST}EE" "$IMGFILE"
+[ -n "${PROTECTIVE_MBR_LIST}" ] && sgdisk -h"${PROTECTIVE_MBR_LIST}EE" "$IMGFILE"
```

Comment:
Shouldn't we address the calling site of where PROTECTIVE_MBR_LIST is set/not set @giggsoff ?